requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+pytest-cov
```

Comment:
This extends requirements.txt, so add `-r requirements.txt` to the top of the file

You need to create a `Dockerfile.dev` that adds these requirements to a dev docker container